File: fizzbuzz.py
"""
	@Author:	Leeroy P. Williams
	@Date:		15/05/2019
	@Description:	Should display Fizz for multiples of 3 and
			Buzz for multiples of 5, then display FizzBuzz
			for multiples of both
"""

x = [x for x in range(1,101)]		# List comprehensions are much faster than regular list appending

for i in x:
	if (i%3 == 0) and (i%5 == 0):
		print("FizzBuzz")
	elif i%3 == 0:
		print("Fizz")
	elif i%5 == 0:
		print("Buzz")
	else:
		print(i)

File: fizzbuzz.go
// @Author:		Leeroy P. Williams
// @Date:		20/05/19
// @Desc:		Should display Fizz for multiples of3 & Buzz for multiples of 5
//				then display FizzBuzz for multiples of both

package main

import "fmt"

func main(){
	
	for i := 100; i > 0; i--{
		if (i%3==0) && (i%5==0){
			fmt.Println("FizzBuzz")
		}else if i%3==0{
			fmt.Println("Fizz")
		}else if i%5==0{
			fmt.Println("Buzz")
		}else{
		fmt.Println(i)
	}
	}
}